django_rq/views.py

```diff
@@ -58,6 +59,28 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _collect_workers_per_configuration(queue_list):
```

Comment:
This is an excellent optimization and should really help if you have many queues in the same Redis DB.